File: ContactBook/app/src/main/java/com/example/contactbook/Model/Person.java
package com.example.contactbook.Model;

import androidx.annotation.NonNull;

public class Person {
    public String id;
    public String firstName;
    public String lastName;

    public Person(String id, String firstName, String lastName) {
        this.id = id;
        this.firstName = firstName;
        this.lastName = lastName;
    }

    @NonNull
    @Override
    public String toString() {
        return firstName + " " + lastName;
    }

    @NonNull
    public String toStringAllInfo() {
        return "Person{" +
                "id=" + id +
                ", firstname=" + firstName + "\"" +
                ", lastname=" + lastName + "\"" +
                "}";
    }

}

File: DBDemo/app/src/main/java/com/example/dbdemo/Person.kt
package com.example.dbdemo

class Person {
    var id: Int = 0
    var firstName: String = ""
    var lastName: String = ""


    constructor(firstName: String, lastName: String) : this(0, firstName, lastName ) {

    }

    constructor(id: Int, firstName: String, lastName: String) {
        this.id = id
        this.firstName = firstName
        this.lastName = lastName

    }
}